src/main/java/solutions/Morze.java

```diff
@@ -0,0 +1,25 @@
+package solutions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Morze {
+    public static void main(String[] args) {
+        Map <Character, String> map = new HashMap<Character, String>();
+        String morze = "england";
+        map.put('a', ".-");
+        map.put('e', ".");
+        map.put('g', "--.");
+        map.put('d', "-..");
+        map.put('l', ".---");
+        map.put('n', "-");
+
+
+        for (int i = 0; i < morze.length(); i ++){
+            morze.charAt(i);
+
+            String str = map.get(morze.charAt(i));
+            System.out.print(" " + str);
```

Comment:
Good

But let's not print everytime string in a loop. Use `StringBuilder` and its method `append`

Reply:
Here you can find an example how to use `StringBuilder` https://www.geeksforgeeks.org/stringbuilder-class-in-java-with-examples/